### ISSUE TITLE
Bug Report: Text is too large for asset.

### ISSUE THREAD

**Issue by Beelijah** [FunkinCrew/Funkin#2932]
## Describe the bug
If you get a 4-digit number for the total notes, the text goes off the background asset!
The only thing is that you can get this on a non-modded copy! Take a look at the screenshot!

## To Reproduce
Get the Total Notes or Max Combo counter to over 999.
## Expected behavior
It should have fit on the asset behind it.

## Screenshots/Video
![image](https://github.com/FunkinCrew/Funkin/assets/154290273/a8565ead-84c9-4edf-a70d-80a349c8e05b)

## Desktop

 - OS: Windows 11
 
 - Browser: Edge
 
- Version: 0.4.1

## Additional context
i got the funny cutscene in the photo

**Comment by Hundrec** [comment]
Good catch. A little adjustment to the left should properly deal with four digit numbers.

Though I wonder how they plan to deal with even _larger_ numbers, if at all.

**Comment by Beelijah** [comment]
Good point

**Comment by AbnormalPoof** [comment]
Confirmed

![Image](https://github.com/user-attachments/assets/c596aa7b-ac76-4efa-8b46-443735214985)

**Comment by Beelijah** [comment]
wait a sec this was opened 6 months ago what

**Comment by charlesisfeline** [comment]
> wait a sec this was opened 6 months ago what

it still hasnt been fixed since v0.5.3

**Comment by EliteMasterEric** [comment]
This could probably solved by nudging the total notes and max combo to the left. I'm fine with the issue occurring with 5 digits (which never happens in vanilla and would require a week six times longer than Weekend 1 to encounter).
